gameobjects/ball.cpp

```diff
@@ -3,16 +3,21 @@
 #include "opponent.h"
 #include "player.h"
 #include "rectangle.h"
+#include "timer.h"
+
+#include <spdlog/spdlog.h>
 
 #include <random>
 
 namespace pong
 {
 
 static constexpr float BALL_WIDTH = 20.0f;
-static constexpr float BALL_SPEED_BOUNCE_INCREMENT = 0.5f;
+static constexpr float BALL_SPEED_BOUNCE_INCREMENT = 75.0f;
 static constexpr float Y_STARTING_POSITION_BOUNDS = 500.0f;
 
+static constexpr int BALL_RESET_WAIT_MS = 3000;
```

Comment:
could consider std::chrono

Reply:
Same comment as above, additionally would I have to change my function signature for SetTimeout to take a std::chrono type? Or at least overload it so it could?

Reply:
I think you would likely change the function signature to take the chrono type. The good thing is that you can then pass in ANY type of chrono (chrono<Seconds> for instance) and it would be automatically converted.

This way anyone could set a time in whatever units they wanted, and the interface would automagically make it work

Reply:
Ah! That is much nicer!